src/vm/errors/hint_errors.rs

```diff
@@ -10,10 +10,18 @@ use super::{exec_scope_errors::ExecScopeError, vm_errors::VirtualMachineError};
 pub enum HintError {
     #[error("HintProcessor failed retrieve the compiled data necessary for hint execution")]
     WrongHintData,
-    #[error("Failed to get ids for hint execution")]
-    FailedToGetIds,
-    #[error("Tried to compute an address but there was no register in the reference.")]
-    NoRegisterInReference,
+    #[error("Unknown identifier {0}")]
+    UnknownIdentifier(String),
+    #[error("Expected ids.{0} to be an Integer value")]
+    IdentifierNotInteger(String),
+    #[error("Expected ids.{0} to be a Relocatable value")]
+    IdentifierNotRelocatable(String),
```

Comment:
It may be a good idea (or not, I'm not really sure) to include the resolved address.

Reply:
On it!

Reply:
Done